src/main/scala/extracells/inventory/MEPassThrough.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * This file is part of Applied Energistics 2.
+ * Copyright (c) 2013 - 2014, AlgorithmX2, All rights reserved.
+ *
+ * Applied Energistics 2 is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Applied Energistics 2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Applied Energistics 2.  If not, see <http://www.gnu.org/licenses/lgpl>.
+ */
+
+package extracells.inventory;
+
+
+import appeng.api.config.AccessRestriction;
+import appeng.api.config.Actionable;
+import appeng.api.networking.security.BaseActionSource;
+import appeng.api.storage.IMEInventory;
+import appeng.api.storage.IMEInventoryHandler;
+import appeng.api.storage.StorageChannel;
+import appeng.api.storage.data.IAEStack;
+import appeng.api.storage.data.IItemList;
+
+
+public class MEPassThrough<T extends IAEStack<T>> implements IMEInventoryHandler<T>
```

Comment:
I don't think you actually need a passthough handler, unless you plan to implement something like a special storage bus support for external storages containing over 2^32 items (and I don't think there is an external storage handler duplicated in EC2)

Reply:
Every storage is a kind of a cell, but passthrough is only useful for a storage bus

Reply:
Removing the PassThrough doesn't seem very straightforward and I am hesitant to do so as the basic AE2 cells implement it in the exact same way.

Reply:
Basic cells of course don't use passthrough handler, they just use generic handler that is also used for storage buses, formation planes, etc.
Well, it may be left in place in case someone would some day use the `MEAdvancedInventoryHandler` for some special storage bus..